src/devices/Arduino/SupportedMode.cs

```diff
@@ -0,0 +1,69 @@
+﻿namespace Iot.Device.Arduino
+{
+    /// <summary>
+    /// Mode bits for the Firmata protocol.
+    /// These are used both for capability reporting as well as to set a mode
+    /// </summary>
+    public enum SupportedMode
+    {
+        /// <summary>
+        /// The pin supports digital input
+        /// </summary>
+        DIGITAL_INPUT = (0x00),
```

Comment:
I know it's ok to have all uppercase names in case of registers. For this specific enum, I feel like we can benefit of having a normal naming.

Reply:
I agree with @Ellerbach here. This enum is public and it should follow the naming convention. The only exception for that we have in dotnet/runtime is enum with 100+ values where values from spec are hard to remember and people frequently copy & paste them

Reply:
I agree. I'll change it.